src/lime/tools/XCodeHelper.hx

```diff
@@ -54,6 +54,21 @@ class XCodeHelper
 		return id;
 	}
 
+	public static function getBootedSimulatorID():String 
+	{
+		// attempts to find a simulator which is currently booted and running
+		var result = System.runProcess("", "xcrun", ["simctl", "list", "devices", "booted"]);
+		var lines = result.split("\n");
+		for (line in lines) {
+			// Match a line like: "    iPhone 15 Pro (E5B048F6-BE70-498E-B5F6-B84B8594DDBF) (Booted)"
+			var match = ~/.*\(([A-F0-9\-]{36})\)\s+\(Booted\)/;
+			if (match.match(line)) {
+				return match.matched(1);
```

Comment:
For consistency with the rest of the code in this class, can you tweak this to use the existing `getSimulatorID()` method that we're using elsewhere? You can probably simplify the initial match as `line.indexOf("(Booted)")` in that case.